doc/source/users_guide/grids.rst

```diff
@@ -176,8 +176,9 @@ The steps for adding a new component grid to the model system follow. This proce
    .. note:: If the ocean and land grids *are identical* then the mapping file will simply be unity and the land fraction will be one minus the ocean fraction.
 
 5. If you are adding a new ocn or rtm grid, create a new rtm->ocn mapping file. (Otherwise you can skip this step.)
-   The process for mapping from the runoff grid to the ocean grid is currently undergoing many changes.
-   At this time, if you are running with a new ocean or runoff grid, please contact Michael Levy (mlevy_AT_ucar_DOT_edu) for assistance. If you are running with standard ocean and runoff grids, the mapping file should already exist and you do not need to generate it.
+   The process for mapping from the runoff grid to the ocean grid is complicated.
+   If you are running with a new ocean or runoff grid and need help, please open a `CIME issue ticket <https://github.com/ESMCI/cime/issues>`_ or make a new post on CESM bulletin board (under the `appropriate ocean model <https://bb.cgd.ucar.edu/cesm/#ocean.147>`_).
```

Comment:
I think that you should just promote the forum - cime tickets should be limited to problems in cime and should minimize problems with RTFM.